go/worker/common/committee/node.go

```diff
@@ -956,6 +956,7 @@ func NewNode(
 
 	// Register transaction sync service.
 	p2pHost.RegisterProtocolServer(txsync.NewServer(chainContext, runtime.ID(), n.TxPool))
```

Comment:
Should we change the server to accept protocol ID or to return it via function call, so that we don't create it twice in the next step?